src/components/Settings/People/DeleteConfirmationDialog.js

```diff
@@ -31,6 +31,34 @@ const DeleteConfirmationDialog = ({
   isDeletingAccount,
   errorDeletingAccount
 }) => {
+  const [requiredConfirmationText, setRequiredConfirmationText] = useState({});
+  const [confirmationText, setConfirmationText] = useState('');
+  const [showConfirmationInput, setShowConfirmationInput] = useState(false);
+
+  useEffect(() => {
```

Comment:
This useEffect is not neccesary. I will leave on conclusion how you can deal with this.

Reply:
I used added a function called handleCloseDialog that resets the confirmation text